src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        String dash = "____________________________________________________________";
+        String greeting1 = "Pai Kia Bot: Eh harlo! Call me Pai Kia Bot.";
+        String greeting2 = "Pai Kia Bot: What you want?";
+        String greeting3 = "Pai Kia Bot: I will keep a list of your responses ah";
+        String farewell = "Pai Kia Bot: Leave so soon ah? Limpeh sleep first, if got no issue don't disturb me.";
+
+        ArrayList<CheckList> inputList = new ArrayList<>();
```

Comment:
Maybe a plural form to represent the collection of Tasks?
Also perhaps a more intuitive name for a Task, instead of a CheckList, which might give the impression that it is a collection of objects?